tests/group_vars/apache/main.yml

```diff
@@ -1,2 +1,33 @@
 ---
 apache_modjk_install: true
+
+apache_extra_libs:
+  - zlib1g-dev
+apache_configure_extra_params: "--enable-deflate"
```

Comment:
Would it be better to use a list in YAML to specify the extra_params?

Reply:
I thought about it, but the `configure` command (https://github.com/idealista/apache_httpd-role/pull/19/files#diff-468007bd6f59ca63e12c1a3613323a17R45) needs to be a one-liner, so using `with_items` won't work if there are more than one `extra_params` (it will execute with only one param each time, so all but the last one won't be used). And as far as I know, for loops (to generate the line) cannot be used in this case.

So my solution is to specify in one line the `extra_params`. I'm open for different approaches, of course.

Reply:
What if you add a test to assert that the deflate module is really enabled?

Reply:
Done